lib/charms/layer/apache_bigtop_base.py

```diff
@@ -491,6 +491,31 @@ def get_hadoop_version():
     return parts[1]
 
 
+def get_package_version(pkg=None):
```

Comment:
Good point -- no one should call this method without a pkg name.

However, i do still want to make sure the value is truthy.  If someone accidentally called this with an empty string, dpkg would happily return the version for every package on the system.